pkg/cmd/pulumi/deployment_settings_config.go

```diff
@@ -492,28 +530,26 @@ func configureGit(ctx context.Context, displayOpts *display.Options, be backend.
 	return nil
 }
 
-func configureGitHubRepo(displayOpts *display.Options,
-	sd *workspace.ProjectStackDeployment, vcsInfo *gitutil.VCSInfo,
-) error {
+func configureGitHubRepo(d *deploymentSettingsCommandDependencies, vcsInfo *gitutil.VCSInfo) error {
+	sd := d.Deployment
+
 	sd.DeploymentSettings.GitHub = &apitype.DeploymentSettingsGitHub{
 		Repository: fmt.Sprintf("%s/%s", vcsInfo.Owner, vcsInfo.Repo),
 	}
 
-	var options []string
-	if err := survey.AskOne(&survey.MultiSelect{
-		Message: "What kind of authentication should it use?",
-		Options: []string{
+	options := promptUserMultiSkippable(
+		d.Yes,
+		"What kind of authentication should it use?",
```

Comment:
```suggestion
		"What kind of authentication should the Pulumi GitHub App use?",
```
same below, just so "it" isn't a bit ambiguous/confusing

Reply:
i will rephrase

Reply:
done